snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java

```diff
@@ -653,10 +653,42 @@ public void import37NewOwlParentInSecondImportBatch() {
   			.body("statedAncestorIds", equalTo(List.of(IComponent.ROOT_ID, Concepts.ROOT_CONCEPT, "404684003")));
   		
   	}
-	
-	private void validateBranchHeadtimestampUpdate(IBranchPath branch, String importArchiveFileName,
+  	
+  	@Test
+	public void import38SnapshotContentWithUnpublishedReferences() throws Exception {
+  		importAndValidateBranchHeadTimestampUpdate(branchPath, "SnomedCT_RF2Release_INT_20180223_content_w_and_wo_effective_time_mix.zip", false, Rf2ReleaseType.SNAPSHOT);
+	}
+  	
+  	@Test
+	public void import39SnapshotContentWithUnpublishedReferences() throws Exception {
```

Comment:
Shouldn't this be delta? The name is the same as above only the number is different.